bind/config.sls

```diff
@@ -169,7 +169,7 @@ bind_rndc_client_config:
 {% endif %}
 
 {% for zone, zone_data in salt['pillar.get']('bind:configured_zones', {}).items() -%}
-{%- set file = salt['pillar.get']("bind:available_zones:" + zone + ":file", zone_data.get('file')) %}
+{%- set file = salt['pillar.get']("bind:available_zones:" + zone + ":file") %}
```

Comment:
To prevent [this conditional](https://github.com/saltstack-formulas/bind-formula/blob/master/bind/config.sls#L178) from failing with a `undefined variable` error, I'd rather change this line to

```yaml
{%- set file = salt['pillar.get']("bind:available_zones:" + zone + ":file"), false %}
```

wdyt?